modules/go-release-notes-1-3-1.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-3-1_{context}"]
+= Release notes for {gitops-title} 1.3.1
+
+{gitops-title} 1.3.1 is now available on {product-title} 4.7, 4.8, and 4.9.
+
+[id="fixed-issues-1-3-1_{context}"]
+== Fixed issues
+
+* If you upgrade to v1.3.0, the Operator does not return an ordered slice of environment variables. As a result, the reconciler fails causing the frequent recreation of Argo CD pods in {gitops-title} clusters running behind a proxy. This update fixes the issue so that Argo CD pods are not recreated. link:https://issues.redhat.com/browse/GITOPS-1489[GITOPS-1489]
```

Comment:
```suggestion
* If you upgrade to v1.3.0, the Operator does not return an ordered slice of environment variables. As a result, the reconciler fails causing the frequent recreation of Argo CD pods in {product-title} clusters running behind a proxy. This update fixes the issue so that Argo CD pods are not recreated. link:https://issues.redhat.com/browse/GITOPS-1489[GITOPS-1489]
```